packages/flarevm.installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -60,7 +60,7 @@ try {
         }
     }
 
-    $installedPackages = chocolatey list -l -r -all | Out-String
+    $installedPackages = choco list -l -r -all | Out-String
```

Comment:
```suggestion
    $installedPackages = choco list -r -all | Out-String
```

There was another change in Chocolatey v2.0.0 where the list command has been made to only list local packages, and the `-l` option is no longer required, or supported.  If this VM is going to be using v2.0.0 of Chocolatey, I would suggest that you remove the `-l` option.